lib/system.nim

```diff
@@ -1087,32 +1087,7 @@ proc align(address, alignment: int): int =
     result = (address + (alignment - 1)) and not (alignment - 1)
 
 when defined(nimNoQuit):
-  proc quit*(errorcode: int = QuitSuccess) = discard "ignoring quit"
```

Comment:
pre-existing documentation regression: https://github.com/nim-lang/Nim/commit/ce44cf03cc4a78741c423b2b3963b48b6d9e6755 I will fix it in the following PR.